client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/InvestigationTableFooter.tsx

```diff
@@ -58,6 +68,15 @@ const InvestigationTableFooter: React.FC<Props> = React.forwardRef((props: Props
         }
     ]
 
+    useEffect(() => {
+        let investigationsToGroup : InvestigationTableRow[] = [];
+        checkedRowsIds.forEach((checkedRowId: number) => {
+            const investigationToGroup = tableRows.find((tableRow: InvestigationTableRow) => tableRow.epidemiologyNumber === checkedRowId);
+            investigationToGroup && investigationsToGroup.push(investigationToGroup);
+        })
+        setInvestigationsToGroup(investigationsToGroup);
+    }, [checkedRowsIds]);
```

Comment:
I don't know if it's an overkill but maybe you should pass down the checked investigations and not only the ids...